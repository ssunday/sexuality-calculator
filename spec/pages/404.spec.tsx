import React from "react";
import NotFound from "../../src/pages/404";
import { render } from "@testing-library/react";

describe("NotFound/404", () => {
  it("matches the snapshot", () => {
    const { container } = render(<NotFound />);

    expect(container).toMatchSnapshot();
  });
});
