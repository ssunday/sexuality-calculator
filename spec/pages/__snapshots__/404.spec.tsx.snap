// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound/404 matches the snapshot 1`] = `
<div>
  <h1>
    404 - Not Found
  </h1>
</div>
`;
